pkg/Microsoft.NETCore.Platforms/readme.md

```diff
@@ -0,0 +1,104 @@
+As such, whenever we want to put a new RID in a project.json in order to get assets specific for that RID we have to define the rid in some package.  Typically that package is `Microsoft.NETCore.Platforms` if the RID is "official".  If you'd like to prototype you can put the RID in any other package and so long as that package is referenced you can use that RID.
+
+###Do I really need to add a new RID?
+If you're prototyping on a platform that is compatible with an existing platform then you can reuse the RID for that exsisting platform.  New RIDs are only needed when an asset needs to be different on a particular platform.
+
+`Microsoft.NETCore.Platforms` attempts to define all RIDs that packages may need, and as such will define RIDs for platforms that we don't actually cross compile for.  This is to support higher-level packages, 3rd party packages, that may need to cross-compile for that RID.
+
+###Naming convention
+We use the following convention in all newly-defined RIDs.  Some RIDs (win7-x64, win8-x64) predate this convention and don't follow it, but all new RIDs should follow it.
+`[os name].[version]-[architecture]-[additional qualifiers]`, for example `osx.10.10-x64` or `ubuntu.14.04-x64`
+- `[os name]` can contain any characters other than `.`
+- `[version]` can contain any characters other than `-`.  Typically a numeric version like 14.04 or 10.0.
+- `[architecture]` can contain any characters other than `-`. Typically: `x86`, `x64`, `arm`, `arm64`
+- `[additional qualifiers]` can be things like `aot`.  Used to further differentiate different platforms.
+
+###Import convention
+Imports should be used when the added RID is considered compatible with an existing RID.
+1. Architecture-specific RIDs should first import the architecture-less RID.  EG: `ubuntu.14.10-x64` should first import `ubuntu.14.10`.
+2. Architecture-specific RIDs that are compatible with a previous version RID for the same OS should then import the previous version, architecture specific RID.  EG: `ubuntu.14.10-x64` should then import `ubuntu.14.04-x64`.  If there is no earlier compatible/supported version, then a versionless RID should be imported.  EG: `ubuntu14.04-x64` should import `ubuntu-x64`.
+3. Architecture-less RIDs that are compatible with a previous version RID for the same OS should then import the previous version, architecture neutral RID.  EG: `ubuntu.14.10` should import `ubuntu.14.04`. If there is no earlier compatible/supported version, then a versionless RID should be imported.  EG: `ubuntu14.04` should import `ubuntu`.
+4. Version-less RIDs shoudld import an OS category.  EG: `ubuntu-x64` should import `linux-x64`, `ubuntu` should import `linux`.
```

Comment:
Do we want to be explicit about how we handle binary compatible distros like Oracle Linux or CentOS which are rebuilds of others (e.g. centos.7 parents to rhel.7)  I think we do a similar thing for Linux Mint?